src/core/styles/_constants.scss

```diff
@@ -111,7 +114,15 @@ $font-size-s: 16px;
 $font-size-m: 18px;
 $font-size-l: 20px;
 $font-size-xl: 24px;
-$font-size-xxl: 36px;
+$font-size-2xl: 28px;
+$font-size-3xl: 32px;
+$font-size-4xl: 36px;
+$font-size-5xl: 40px;
+$font-size-6xl: 44px;
+$font-size-7xl: 52px;
+$font-size-8xl: 60px;
```

Comment:
I also suggest reducing the number of different fonts in favor of standard ones and making large values ​​multiples of `16px`:
`44px --> 40 or 48px`
`52px --> 48px`
`60px --> 64px`